### PR TITLE
Added clarifications to WaveLoss helpfile

### PR DESCRIPTION
**Pull request by LFSaw**

### CODE REVIEW

File: source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp [hunk 1]
```diff
@@ -4,34 +4,34 @@ categories:: UGens>Filters>Nonlinear
 related:: Classes/Squiz
 
 DESCRIPTION::
-Uses the zero-crossings to divide an audio stream into tiny segments, and simply discards a fraction of the segments (replacing them with silence). The technique was described in a lecture by Trevor Wishart.
+Divide an audio stream by zero crossings of the signal into tiny segments and discard a fraction of them (i.e. replace them with silence of the same length).
```
Comment (danstowell) [inline source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp]: Do you really find this sentence clearer? I find it less clear, and sorry to be rude but I guess it's quite a non-native sentence construction to put "by zero crossings" in the middle like that.

File: source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp [hunk 2]
```diff
@@ -4,34 +4,34 @@ categories:: UGens>Filters>Nonlinear
...
-Parameters: the filter drops strong::drop:: out of strong::outof:: chunks. strong::mode:: can be 1 to drop chunks in a simple deterministic fashion (e.g. always dropping the first 30 out of a set of 40 segments), or 2 to drop chunks randomly but in an appropriate proportion.
+Parameters: the filter drops strong::drop:: out of strong::outof:: chunks. strong::mode:: can be code::1:: to drop chunks in a simple deterministic fashion (e.g. always dropping the first code::30:: out of a set of code::40:: segments), or code::2:: to drop chunks randomly but in an appropriate proportion.
 
 CLASSMETHODS::
 
-METHOD:: ar
+METHOD:: ar, kr
 argument:: in
 argument:: drop
+number of waves to drop within a code::outof:: waves
```
Comment (danstowell) [inline source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp]: I suggest "number of waves to drop within each region of code::outof:: waves"

File: source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp (hunk 1 above)
Comment (LFSaw) [inline source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp]: fair enough. any constructive ideas?

Comment (danstowell) [inline source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp]: The original sentence reads fine to me, so all I can really say is keep it the same. If it's troublesome for other readers then I can happily react to suggestions...

Comment (LFSaw) [inline source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp]: "Uses the zero-crossings to divide an audio stream into tiny segments" – does implicietly refer to the zero-crossings but does not tell that it is divided _at_ those zero-crossings.

this is what made me wonder and rephrase.

Comment (LFSaw) [inline source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp]: ... most of the time, in most languages at least.

Comment (danstowell) [inline source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp]: OK I see. How about

"Divide an audio stream into tiny segments, using the signal's zero-crossings as segment boundaries, and discard a fraction of them (i.e. replace them with silence of the same length)."

Comment (LFSaw) [inline source/MCLDUGens/sc/HelpSource/Classes/WaveLoss.schelp]: super! Thanks!

**Pull request merged**